src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -117,7 +117,7 @@
     {% endfor %}
 
     <div class="sf-tabs">
-        <div class="tab">
+        <div class="tab {{ collector.countMissings == 0 ? 'tab-active' }}">
```

Comment:
Wouldn't `tab{{ collector.countMissings == 0 ? ' tab-active' }}` be more optimal for output?

Reply:
We don't care about redundant spaces. They don't hurt at all. And this code is more readable in the template.

Reply:
@Destroy666x you are right, but as @stof said, we always prioritize the readability of Twig templates over the generated HTML code. So let's keep it.

Reply:
Ok, I see. IMO both versions are equally readable though